include/Core/Terms/Modules/DeleteTermsByNameModule.php

```diff
@@ -62,31 +62,32 @@ protected function get_term_ids_to_delete( $options ) {
 		$term_ids = array();
 		$value    = $options['value'];
 		$operator = $options['operator'];
-
-		switch ( $operator ) {
-			case 'equal_to':
-				$term_ids = $this->get_terms_that_are_equal_to( $value, $options );
-				break;
-
-			case 'not_equal_to':
-				$term_ids = $this->get_terms_that_are_not_equal_to( $value, $options );
-				break;
-
-			case 'starts_with':
-				$term_ids = $this->get_terms_that_starts_with( $value, $options );
-				break;
-
-			case 'ends_with':
-				$term_ids = $this->get_terms_that_ends_with( $value, $options );
-				break;
-
-			case 'contains':
-				$term_ids = $this->get_terms_that_contains( $value, $options );
-				break;
-
-			case 'not_contains':
-				$term_ids = $this->get_terms_that_not_contains( $value, $options );
-				break;
+		if ( '' !== $value ) {
```

Comment:
Let's use the `empty` function instead of checking for empty string.

Also if `$value` is empty, immediately return an empty array and exit the function.